### PR TITLE
Seccomp Update the existing one and add weaker config

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: seccomp.go [hunk 1]
```diff
@@ -1,1367 +1,1602 @@
...
-		{
-			"name": "personality",
-			"action": "SCMP_ACT_ALLOW",
-			"args": [
-				{
-					"index": 0,
-					"value": 0,
-					"valueTwo": 0,
-					"op": "SCMP_CMP_EQ"
+	seccompDefaultProfile = `{
```
Comment (alban) [inline seccomp.go:7]: Could you add a comment with a link saying where this was taken from?

And for seccompWeakProfile, a comment saying what's the difference?

Maybe those two constants should be in different files (seccomp-default.go and seccomp-weak.go), that would make it easier to run `diff` between the two.

Comment (surajssd) [reply seccomp.go:7]: done

**Comment by surajssd** [comment]
I am adding weaker seccomp profile but it will be consumed only when the code will be added. Right now going forward with merging it?

**Comment by indradhanush** [comment]
@surajssd Yes, let's get this merged in favor of velocity. We can come back to iterate and fix issues.

**Review by alban — APPROVED** [review]

**Pull request merged**